app/views/logix/examples.html.erb

```diff
@@ -95,7 +95,18 @@
         </div>
         <br />
 
+        <div class="row feature">
+                 <div class="col-12 col-md-8">
+                     <h3 class="card-title">Universal Shift Register</h4>
+                     <p class="card-text">This is a universal bidirectional shift register which can shift the input to the right as well as left.</p>
```

Comment:
Should be `to the right as well as THE left`

Reply:
👍